### PR TITLE
[deckhouse-controller] Change controller logger

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Description
Improved deckhouse controller logger

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse-controller
type: feature
summary: Improved deckhouse controller logger
impact_level: default
```

### CODE REVIEW

File: deckhouse-controller/pkg/controller/deckhouse-release/controller.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func NewDeckhouseReleaseController(ctx context.Context, mgr manager.Manager, dc
 	moduleManager moduleManager, updateSettings *helpers.DeckhouseSettingsContainer, metricStorage *metric_storage.MetricStorage,
 	preflightCountDown *sync.WaitGroup, logger *log.Logger,
 ) error {
-	lg := logger.With("component", "DeckhouseRelease")
+	lg := log.Default().Named("DeckhouseRelease")
```
Comment (ldmonster) [inline deckhouse-controller/pkg/controller/deckhouse-release/controller.go:83]:
```suggestion
	lg := logger.Named("deckhouse-release-controller")
```

File: deckhouse-controller/pkg/controller/deckhouse-release/client_test.go [hunk 2]
```diff
@@ -95,7 +95,7 @@ func setupControllerSettings(
 	rec := &deckhouseReleaseReconciler{
 		client:         cl,
 		dc:             dc,
-		logger:         log.NewNop(),
+		logger:         log.Default(),
```
Comment (ldmonster) [inline deckhouse-controller/pkg/controller/deckhouse-release/client_test.go:98]: what's purpose of global logger in tests? 
do you check output?

File: deckhouse-controller/pkg/controller/deckhouse-release/controller.go (hunk 1 above)
Comment (ldmonster) [reply deckhouse-controller/pkg/controller/deckhouse-release/controller.go:83]: we already named it when we call NewDeckhouseReleaseController

File: deckhouse-controller/pkg/controller/module-controllers/release/controller.go [hunk 3]
```diff
@@ -110,7 +111,7 @@ func NewModuleReleaseController(
 	preflightCountDown *sync.WaitGroup,
 	logger *log.Logger,
 ) error {
-	lg := logger.With("component", "ModuleReleaseController")
+	lg := log.Default().Named("ModuleReleaseController")
```
Comment (ldmonster) [inline deckhouse-controller/pkg/controller/module-controllers/release/controller.go:114]:
```suggestion
	lg := logger
```
we already named it when we call NewModuleReleaseController

**Review by yalosev — APPROVED** [review]

**Pull request merged**